providers/rtc/RTCConstants.groovy

```diff
@@ -0,0 +1,84 @@
+package rtc
+/**
+ * © Copyright IBM Corporation 2018.
+ * © Copyright PrimeUP Solucoes em TI LTDA 2018.
+ * LICENSE: Apache License, Version 2.0 https://www.apache.org/licenses/LICENSE-2.0
+ */
+
+/**
+ * This class collects all constants used by the RTC provider
+ */
+public class RTCConstants {
+	
+	static final String ROOT_SERVICES = "/rootservices"
+	static final String JSECURITYCHECK = "/j_security_check"
+
+	static final String AUTHREQUIRED_HEADER = "X-com-ibm-team-repository-web-auth-msg"
+	static final String AUTHREQUIRED_VALUE = "authrequired"
+	static final String AUTHFAILED_VALUE = "authfailed"
+	static final String BASIC_HEADER = "WWW-Authenticate"
+	static final String AUTHORIZATION_HEADER = "Authorization"
+	static final String APPLICATION_RDF_XML_TYPE = "application/rdf+xml"
+	static final String ACCEPT_HEADER = "Accept"
+	static final String CONTENT_TYPE_HEADER = "Content-Type"
+	static final String OSLC_CORE_VERSION_HEADER = "OSLC-Core-Version"
+	static final String OSLC_VERSION = "2.0"
+	static final String FILED_AGAINST_PROPERTY = "filedAgainst"
+	static final String FILE_NAME = "IssueDetails-%s.html"
+	static final String ATTACHMENT_UPLOAD_URL = "%s/service/com.ibm.team.workitem.service.internal.rest.IAttachmentRestService/?projectId=%s&multiple=true"
+
+	//Provider name
+	static final String PROVIDER_NAME = "rtc"
+
+	//Required fields
+	static final String SERVER_URL = "url"
+	static final String USERNAME = "username"
+	static final String PASSWORD = "password"
+	static final String PROJECTAREA = "projectarea"
+	static final String ISSUETYPE = "issuetype"
+	
+	//Optional fields	
+	static final String OTHERFIELDS = "otherfields"
+	static final String FILEDAGAINST = "filedAgainst"
+	static final String SUMMARY = "summary"
+	static final String DESCRIPTION = "description"
+	
+	//Description
+	static final String PROVIDER_DESCRIPTION =

```

Comment:
Can you change these to use the 'def' keyword rather than 'String" like in the other providers?  There is an issue right now where the providers GET endpoint isn't working due PROVIDER_DESCRIPTION being a String and not a String[]. Changing these to 'def' will fix the issue.